src/Umbraco.Web.Common/DependencyInjection/UmbracoBuilder.ImageSharp.cs

```diff
@@ -50,7 +51,9 @@ public static IServiceCollection AddUmbracoImageSharp(this IUmbracoBuilder build
                 })
                 .SetCache<PhysicalFileSystemCache>()
                 .SetCacheHash<CacheHash>()
-                .AddProvider<PhysicalFileSystemProvider>()
+                .ClearProcessors()
+                .RemoveProcessor<ResizeWebProcessor>() // The Resize processor is added by default, so remove it to ensure that the crop processor runs first
```

Comment:
Unnecessary.

Reply:
If the order of adding processors doesn't matter and the default ones are already added by `AddImageSharp()`, it's unnecessary to clear and explicitly add these... So we should only add our custom processor here.